events/makernight.md

```diff
@@ -1,28 +0,0 @@
----
-title: Maker Night
-parent: "Events & Classes"
-image: /events/makernight.jpg
-layout: default
-permalink: /events/makernight/
```

Comment:
I think I left this in place before because there could be things linking to it. Maybe just worth redirecting to the what's on page?

Reply:
Are you thinking there might be external things linking to it? Happy to keep it anyway, to not contribute to the link rot of the internet...

Reply:
That's what I was thinking -- but also in the previous PR where I justified leaving it in, I happily ripped out other pages without a care in the world 🤷 Not strongly opinionated, whatever makes more sense to you!